astro-client/astro.go

```diff
@@ -26,7 +26,7 @@ type Client interface {
 	ReportDagDeploymentStatus(input *ReportDagDeploymentStatusInput) (DagDeploymentStatus, error)
 	// Image
 	CreateImage(input CreateImageInput) (*Image, error)
-	DeployImage(input DeployImageInput) (*Image, error)
+	DeployImage(input *DeployImageInput) (*Image, error)
```

Comment:
nit: trying to understand why did we move to the pointer?

Reply:
It was giving me a lint error

`astro-client/astro.go:180:34: hugeParam: input is heavy (88 bytes); consider passing it by pointer (gocritic)`

Reply:
ahh ok